test/parallel/test-cluster-worker-init.js

```diff
@@ -3,30 +3,25 @@
 // verifies that, when a child process is forked, the cluster.worker
 // object can receive messages as expected
 
-require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var msg = 'foo';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const msg = 'foo';
 
 if (cluster.isMaster) {
-  var worker = cluster.fork();
-  var timer = setTimeout(function() {
-    assert(false, 'message not received');
-  }, 5000);
+  const worker = cluster.fork();
 
-  timer.unref();
-
-  worker.on('message', function(message) {
+  worker.on('message', common.mustCall((message) => {
     assert(message, 'did not receive expected message');
```

Comment:
Sure thing. It will be in the format
```
assert.strictEqual(message, true, 'did not receive expected message');
```
because worker sends `true` as message value.